### ISSUE TITLE
Missing Actions for Inherited/Extended Controllers 

### ISSUE THREAD

**Issue by richjdsmith** [marcoroth/stimulus-lsp#198]
I am using the wonderful [tailwindcss-stimulus-components ](https://github.com/excid3/tailwindcss-stimulus-components) from @excid3 in a project. I am extending them by importing the specific component, and then extending. Here is an example of a search_dropdown_controller

```
import { Dropdown } from "tailwindcss-stimulus-components"
// import { transition } from 'tailwindcss-stimulus-components'

export default class extends Dropdown {
  static targets = ["searchGlass"]

  openValueChanged() {
    super.openValueChanged();
    this.searchGlassTarget.classList.toggle(['rotate-90'], this.openValue)
  }



}

```

Unfortunately, the Stimulus-LSP seems to be unable to recognize that this controller is inheriting classes from the parent `Dropdown` controller, and filling my errors output with various **"XXX" isn't a valid Controller Action on the "search-dropdown" controller. Did you mean "openValueChanged"?**

I am not sure if I am just configuring it incorrectly, or if this is a missing feature, but I wanted to bring it to attention. 

Thanks and sorry if my explanation is missing anything!

**Comment by marcoroth** [comment]
Hey @richjdsmith, thanks for reporting!

I just added support for this exact use-case in the `stimulus-parser` library (which is used to build this LSP). See https://github.com/marcoroth/stimulus-parser/releases/tag/v0.1.0

The next Stimulus LSP version is going to ship this. This should then also resolve the wrong diagnostics you are seeing now.

Sorry for the inconvenience!

**Comment by marcoroth** [comment]
This should be resolved with v1.0.0, please re-open this issue if you still see something unexpected! Thank you!

**Comment by excid3** [comment]
Awesome, will give it a try again!
